llama_hub/file/image_deplot/README.md

```diff
@@ -0,0 +1,17 @@
+# Image Loader (Blip2)
+
+This loader captions an image file containing a tabular graph (bar chart, line charts) using deplot.
+
+## Usage
+
+To use this loader, you need to pass in a `Path` to a local file.
+
+```python
+from pathlib import Path
+from llama_index import download_loader
```

Comment:
nit: we're deprecating the download_loader usage  - can still keep but we'd want to highlight `from llama_index.file.image_deplot.base import ImageTabularGraphReader`

Reply:
Are we going to deprecate this repo and the source of truth will be on llama_index?

Reply:
Or is it just a typo? e.g. `from llama_hub.file.image_deplot.base import ImageTabularGraphReader`

Reply:
oh oops yeah that's what i meant